src/sagemaker_algorithm_toolkit/hyperparameter_validation.py

```diff
@@ -243,16 +244,34 @@ def _visit(name, visited, stack):
 
         return dependencies_stack
 
+    def _declare_alias(self, key_name, alias_name):
```

Comment:
by convention, I would not prefix this method with an `_` as this is not a helper method and is invoked outside of the class.